### ISSUE TITLE
Windump makes weird error

### ISSUE THREAD

**Issue by gpotter2** [hsluoyz/WinDump#1]
With the winpcap version, we could do:

`windump -r -`

Then we passed some data with a python program.
I used to get:

```
reading from file -, link-type RAW (Raw IP)
15:16:47.666223 IP 127.0.0.1.20 > 127.0.0.1.80: S 0:0(0) win 8192
15:16:47.666975 IP 127.0.0.1.53 > 127.0.0.1.53: [|domain]
15:16:47.667897 IP 127.0.0.1 > 127.0.0.1: ICMP echo request, id 0, seq 0, length 8
```

When I use this version, I get:

```
reading from file -, link-type RAW (Raw IP)
C:\WINDOWS\system32\windump.exe: Sending packets isn't supported on savefiles
```

Any ideas?

Thank you !

**Comment by hsluoyz** [comment]
Can you provide the specified file in the ``windump -r`` command?

**Comment by hsluoyz** [comment]
@gpotter2 , this bug has been fixed in latest WinDump 0.2, please try it at: https://github.com/hsluoyz/WinDump/releases

**Comment by gpotter2** [comment]
That did the trick, thanks ! 

I've noticed another small bug:

on the winpcap version, performing only:
`windump`
will start sniffing on the main interface.

On this version, it shows:

> C:\Users\gpotter>windump
> windump: listening on \Device\NPF_{C56DFF[...]}
> windump: Error opening adapter: Le pÚriphÚrique spÚcifiÚ est introuvable. (20)

(translation: the specified adapter does not exist)

**Comment by hsluoyz** [comment]
It seems to be a bug of Libpcap (the wpcap.dll API of Npcap). I have reported it in their issues here: https://github.com/the-tcpdump-group/libpcap/issues/562

**Comment by gpotter2** [comment]
Great, Thanks !

**Comment by hsluoyz** [comment]
Hi @gpotter2 ,

the default adapter issue has been fixed in latest WinDump 0.3, please try it at: https://github.com/hsluoyz/WinDump/releases

**Comment by gpotter2** [comment]
Thank you !
